Project 3/block_quantize.py

```diff
@@ -69,9 +73,13 @@ def quantize(frame_data, n):
         for block_x in range(0, len(frame), 8):
             for block_y in range(0, len(frame[block_x]), 8):
                 block = frame[block_x:block_x+8, block_y:block_y+8]
-                result.extend(quantize_block(block, n, frame_num, block_x, block_y))
+                block_dict,block_hist = quantize_block(block, n, frame_num, block_x, block_y)
+                block_hist_dict[block_x,block_y] = block_hist
+                result.extend(block_dict)
+
+            frame_block_dict[frame_num] = block_hist_dict
 
```

Comment:
This is where we add the histogram to the dictionary for the histograms of the blocks where the coordinates are the key. We then add that dictionary to the frame dictionary where the frame number is the key to access the dictionaries of the regions.